chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ProductController.java

```diff
@@ -324,8 +324,10 @@
 
             for (CertifiedProductDetailsDTO filterCp : filteredResults) {
                 //Add it to the list of duplicates
-                duplicateChplProductNumbers.add(new DuplicateChplProdNumber(
-                    cpDTO.getChplProductNumber(), filterCp.getChplProductNumber(), filterCp.getChplProductNumber()));
+                if (!cpDTO.getChplProductNumber().equals(filterCp.getChplProductNumber())) {
```

Comment:
This is a string comparison, right? Don't we usually do string comparisons with some sort of StringUtil operator in some way? @kekey1 am I crazy?

Reply:
I think you might be thinking of the StringUtils.isEmpty methods that we use (avoids checking for both null and blank strings - and forgetting one or the other of those possibilities - all over the place). I'm pretty sure that for string comparisons we just use equals or equalsIgnoreCase as applicable.